crates/zk-por-core/src/recursive/tests.rs

```diff
@@ -0,0 +1,70 @@
+use super::{
+    super::{
+        account::gen_accounts_with_random_data,
+        merkle_sum_prover::{
+            circuits::merkle_sum_circuit::build_merkle_sum_tree_circuit,
+            prover::MerkleSumTreeProver,
+        },
+        types::{C, F},
+    },
+    circuit::build_recursive_n_circuit,
+};
+
+use super::prove::prove_n_subproofs;
+use plonky2_field::types::Field;
+
+#[ignore]
```

Comment:
can use a feature to ignore; and modify github rust.yml to enable that feature; we may not ignore when running locally

cargo test --all --features "skip-tests"